torchrl/objectives/sac.py

```diff
@@ -378,6 +378,7 @@ def _get_value_v2(self, tensordict, _alpha, actor_params, qval_params):
 
         """
         tensordict = tensordict.clone(False)
+        next_tensordict = step_mdp(tensordict)
```

Comment:
We're doing that below, why is it needed here?

Reply:
Oh, I think that was a leftover when I merged main into the branch. Just fixed it :)